doc/lua_api.txt

```diff
@@ -2165,12 +2151,13 @@ representing the alpha value must (always) be two hexadecimal digits.
 `ColorSpec`
 -----------
 A ColorSpec specifies a 32-bit color.  It can be written in either:
-table form, each element ranging from 0..255 (a, if absent, defaults to 255):
-    `colorspec = {a=255, r=0, g=255, b=0}`
-numerical form, the raw integer value of an ARGB8 quad:
-    `colorspec = 0xFF00FF00`
-or string form, a ColorString (defined above):
-    `colorspec = "green"`
+
+* table form, each element ranging from 0..255 (a, if absent, defaults to 255):
+    * `colorspec = {a=255, r=0, g=255, b=0}`
+* numerical form, the raw integer value of an ARGB8 quad:
+    * `colorspec = 0xFF00FF00`
+* or string form, a ColorString (defined above):
```

Comment:
"or" isn't required for lists, can be omitted to result in

`* string form,`

Reply:
I agree.